src/hoprd.rs

```diff
@@ -53,17 +53,27 @@ pub struct HoprdSpec {
 pub struct HoprdConfig {
     pub announce: Option<bool>,
     pub provider: Option<String>,
+    #[serde(rename(deserialize = "defaultStrategy"))]
```

Comment:
camelCase rename can be done globally on the object and more idiomatically: https://serde.rs/attr-rename.html

Reply:
Modified, Thanks!!!